docs/guide/docs/introduction.md

```diff
@@ -3,35 +3,40 @@ id: introduction
+It is a complete platform that ships with all the tools you need to build, deploy, and manage production-grade bots in record time. Here are some of the components that are installed by default:
 
-- An NLU Engine
-- An administration dashboard
-- A visual flow editor
-- A chat emulator / debugger
-- Support for multiple messaging channels
+- An NLU Engine which understands the meaning and intentions behind user statements even where grammatical mistakes exist.
+- An administration interface that can be viewed from a browser, exposing modules to manage content and show chatbot  statistics, among other things. 
+- A visual workflow builder used for creating conversations and other interactions.
+- A chat emulator/debugger to simulate conversations and inspect the chatbot engine's logic.
+- Support for popular messaging channels like Facebook Messenger, Slack, Telegram, and others.
 
-### Why Botpress
+## Why Botpress
 
 Botpress has many advantages over other platforms and frameworks:
 
-- It runs **fully on-prem**, so you have full control over the data that comes in and out
-- The core of the Botpress platform is **open-source**
-- Extremely **flexible** and developer-friendly
-- **Zero** external dependencies
+- It runs **fully on-premise**, so you can host it on a server of your choice. This gives you full control over the data that comes in and out and helps you secure your data. 
+The Botpress platform's core is **open-source** with thousands of Github contributors and will be free forever.
+- The framework is very **flexible** and developer-friendly because Botpress can be customized primarily using Javascript with capabilities to include code in other languages.
+- Botpress relies on **zero** third-party services, making it the only fully featured Chatbot-Building platform
+
+## How to build a Chatbot
 
-## How to build a bot
+There are three main steps involved in creating chatbots.
 
-There are three main steps involved in creating bots.
+Developers start by **Building** a chatbot, which consists of adding the required messaging channels, integrating the necessary backend components and applications your chatbot will need to interact with, and creating conversation workflows.
 
-Developers start by **Building** the bot, which consists of adding the necessary messaging channels, coding the necessary backend integrations your bot will need to interact with and creating the main dialog flows.
+Once done, you'll need to **Deploy** your chatbot somewhere, which could be on a cloud server or your internal hosting infrastructure.
 
-Once done, you'll need to **Deploy** the bot somewhere, which could be on the cloud or your own infrastructure.
+Finally, you'll give other people access to your chatbot so they can **Manage** it once it is in use. Managing a chatbot entails analyzing the conversations, refining the conversational experience by building new flows, and making it smarter by continuously improving the NLU.
```

Comment:
you use workflows instead of flows pretty much everywhere else. for a matter of consistency let's keep it as workflow